common/guild_base.h

```diff
@@ -5,6 +5,43 @@
 #include <map>
 #include <string>
 #include <vector>
+#include "timer.h"
+#include "../common/repositories/guild_members_repository.h"
+
+#define GOUT(x) out.x = in->x;
```

Comment:
If this macro is going to be used only inside `guild_base.cpp` it should just be placed at the top of that file to avoid leaking the macro to anything that includes this header.

Also not sure we should be using macros just for assignment like this outside of packet encoder stuff but maybe it simplified something

Reply:
Created the routing to copy the info and therefore no longer requiring the macro.  Removed